manifests/database.pp

```diff
@@ -13,16 +13,20 @@
     user     => $gutterball::dbuser,
     password => $gutterball::dbpassword,
   } ~>
+  file { '/usr/bin/gbdb':
+    ensure  => file,
+    content => template('gutterball/gbdb.conf.erb'),
+    mode    => '0777',
```

Comment:
On second thought, if you are storing the password in the script, then maybe this shouldn't be world readable.